### PR TITLE
Change in Ok button text to better represent actual functionality

### LINKED ISSUE #5090: Reword settings dialog button to Safe & Close from Ok
**Justification**
OK is implicitly saving the changes we make. We strive to do this with code already, lets do it with the settings window.

**Description**
Achieve this with Save & Close for the button text to make it explicit and unambiguous what will occur.

**Code location**
Update in the [RubberduckUI.resx](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Resources/RubberduckUI.resx#L174-L176)

### PR DESCRIPTION
**Pull request by GusAV**
Closes #5090

### CODE REVIEW

File: Rubberduck.Resources/RubberduckUI.resx [hunk 1]
```diff
@@ -171,8 +171,8 @@
   <data name="NameLabelText" xml:space="preserve">
     <value>Name:</value>
   </data>
-  <data name="OK" xml:space="preserve">
```
Comment (retailcoder) [inline Rubberduck.Resources/RubberduckUI.resx]: Adding a new resource key is a good idea - renaming the OK key without adjusting the places referencing it is going to break all "Ok" captions though.

I would suggest keeping the OK resource key and adding a new one with the new caption, and then the settings dialog's ok button needs to use the new resource key.

Comment (retailcoder) [reply Rubberduck.Resources/RubberduckUI.resx]: The resource key is referenced here:

https://github.com/rubberduck-vba/Rubberduck/blob/170191673789ec46cbd58519beaf443f42abeefa/Rubberduck.Core/UI/Settings/SettingsControl.xaml#L150

Comment (GusAV) [reply Rubberduck.Resources/RubberduckUI.resx]: Got it, will change that.

Comment (retailcoder) [reply Rubberduck.Resources/RubberduckUI.resx]: Thanks!

Comment (GusAV) [reply Rubberduck.Resources/RubberduckUI.resx]: I believe this should do it.

**Pull request merged**